Xamarin.Forms.Platform.Android/Renderers/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -191,10 +194,7 @@ protected virtual void UpdateFlyoutHeaderBehavior()
 		{
 			var context = _shellContext.AndroidContext;
 
-			Thickness margin = default(Thickness);
-
-			if (_flyoutHeader != null)
-				margin = _flyoutHeader.Margin;
+			var margin = _flyoutHeader?.Margin ?? default;
```

Comment:
@kvpt we don't have  C# 7.3 on our 4.2 branch it's only 7.0 so can you just change this to default(Thickness) ?